lib/sarif/base_sarif.rb

```diff
@@ -0,0 +1,42 @@
+module Sarif
+  class BaseSarif
+    DEFAULT_URI = "https://github.com/coinbase/salus".freeze
+
+    attr_reader :name, :tool, :results, :scan_report
+    def initialize(scan_report)
+      @name = scan_report.scanner_name
+      @scan_report = scan_report
+      @tool = tool_info
+      @results = results_info
+    end
+
+    # Retrieve tool section for sarif report
+    def tool_info
+      {
+        "driver": {
+          "name" => @scan_report.scanner_name,
+          "semanticVersion" => @scan_report.version || "",
+          "informationUri" => DEFAULT_URI
```

Comment:
- `semanticVersion` might not apply to all the scanners, so it might be better to use `version` here
-  `DEFAULT_URI` is a better fit for `run.conversion.tool.driver`